.github/workflows/test.yml

```diff
@@ -67,7 +67,7 @@ jobs:
 
       - name: Run tests
         shell: bash
-        run: yarn test --nmt _Bytecode
+        run: yarn test
```

Comment:
👀

Reply:
ahh you disabled both deployment tests completely, hmmmmmm 🤔

Reply:
They started to fail after testing for the protocol token pausability in `CommonDeploymentTest`.

Reply:
hmm here's the catch, i'd modify `test:local` to ignore `_Bytecode`, but making them `abstract` means that if i wanna run them locally, i cannot w/o modifying the codebase.

yet, since we use that bash sh*t, we cannot do `yarn test:local --mc E2E`, so issa pain to have non-working tests, i'm on a crossover here 🤔

Reply:
Most tests whose fork environment consists of the contracts fetched from the latest deployment fail.

Specifically in Mainnet, all fail but 3.

Specifically in Goerli, none `_Bytecode` ones pass, but `_Params`, `_CParams` and `_Auth` ones do (except for the modified `test_ProtocolToken_Auth`). Of course, the newly added `test_ProtocolToken_Pausable` also fails.

Will the contracts be redeployed on Goerli? Soon? If not, selectively ignoring the test cases that start to fail within each PR doesn't seem like an elegant solution. 😕

By the way, I can run `yarn test:local --mc E2E` and it executes well.